tests/tck/features/fulltext_index/FulltextIndexQuery1.feature

```diff
@@ -1,107 +0,0 @@
-# Copyright (c) 2022 vesoft inc. All rights reserved.
-#
```

Comment:
Why delete these tests? i add more cases in #5575 in these features.

Reply:
Because PREFIX,FUZZY,WILDCARD,REGEXP are all abandoned